python/pfs/drp/stella/datamodel/pfsFiberNorms.py

```diff
@@ -1,77 +1,44 @@
-    def plot(
-        self,
-        pfsConfig: pfs.datamodel.PfsConfig,
-        axes: Optional["matplotlib.axes.Axes"] = None,
-        lower: float = 2.5,
-        upper: float = 2.5,
-        size: float = 10,
-    ) -> Tuple["matplotlib.figure.Figure", "matplotlib.axes.Axes"]:
+    # Add the plot function as a class method.
+    def plot(self,
+             pfsConfig: pfs.datamodel.PfsConfig,
+             axes: Optional["Axes"] = None,
+             lower: float = 2.5,
+             upper: float = 2.5,
+             size: float = 10,
+             title: Optional[str] = None
+             ) -> "Figure":
         """Plot fiber normalization values
 
         Parameters
         ----------
+        fiberNorms : `PfsFiberNorms`
+            Fiber normalization values.
         pfsConfig : `pfs.datamodel.PfsConfig`
             Configuration for the PFS system
         axes : `matplotlib.axes.Axes`, optional
             Axes to plot on; if None, create a new figure and axes.
         lower, upper : `float`
-            Lower and upper bounds for plot, in units of standard deviations.
+            Lower and upper bounds for plot, in units of standard deviations. Default is 2.5.
         size : `float`
-            Size of the markers.
+            Size of the markers. Default is 10.
+        title : `str`, optional
+            Title for the plot.
 
         Returns
         -------
         fig : `matplotlib.figure.Figure`
             Figure containing the plot.
-        axes : `matplotlib.axes.Axes`
-            Axes containing the plot.
         """
-        import matplotlib
-        import matplotlib.pyplot as plt
-        import matplotlib.cm
-        from matplotlib.colors import Normalize
-        from ..fitDistortedDetectorMap import addColorbar
-
-        cmap = matplotlib.cm.coolwarm
-        if axes is None:
-            fig, axes = plt.subplots()
-        else:
-            fig = axes.figure
-
-        pfsConfig = pfsConfig.select(fiberId=self.fiberId)
-        indices = np.argsort(pfsConfig.fiberId)
-        assert np.array_equal(pfsConfig.fiberId[indices], self.fiberId)
-        xx = pfsConfig.pfiCenter[indices, 0]
-        yy = pfsConfig.pfiCenter[indices, 1]
-
-        with warnings.catch_warnings():
-            warnings.filterwarnings("ignore", r"All-NaN (slice|axis) encountered")
-            values = np.nanmedian(self.values, axis=1)
-        good = np.isfinite(values)
-        median = np.median(values[good])
-        rms = robustRms(values[good])
-        lower = max(median - lower*rms, np.nanmin(values))
-        upper = min(median + upper*rms, np.nanmax(values))
-        norm = Normalize(vmin=lower, vmax=upper)
-
-        axes.scatter(xx, yy, marker="o", c=values, cmap=cmap, norm=norm, s=size)
-        axes.set_aspect("equal")
-        addColorbar(fig, axes, cmap, norm, "Fiber normalization")
-
-        return fig, axes
+        from pfs.drp.qa.utils.plotting import plotFiberNorms
```

Comment:
Please restore this code, and leave the definition of the plotting here: drp_stella does not depend on drp_qa.

Reply:
Ah, got it, sorry, it looks like I misunderstood the aim of the PR. I'll clean this up.